data-analysis/data-tools/data-analysis-using-sql/syntax-ii.md

```diff
@@ -121,6 +122,6 @@ FROM cities
 - WHERE
 - `>`
 - AND
-- `=`
+- LIKE
```

Comment:
we don't need to change this to `LIKE` because the filter is looking at an exact match

```suggestion
- =
```